app/models/invenio_rdm_record_converter.rb

```diff
@@ -180,7 +180,7 @@ def invenio_metadata
       "publication_date": format_publication_date(@generic_file.date_created.shift || @generic_file.date_uploaded.to_s),
       "subjects": SUBJECT_SCHEMES.map{ |subject_type| subjects_for_scheme(@generic_file.send(subject_type), subject_type) }.compact.flatten,
       "contributors": contributors(@generic_file.contributor),
-      "dates": @generic_file.date_created.map{ |date| {"date": date, "type": "other", "description": "When the item was originally created."} },
+      "dates": @generic_file.date_created.map{ |date| {"date": date, "type": {"id": "other"}, "description": "When the item was originally created."} },
```

Comment:
Oh this is always the date of creation? Then, we should use this id: https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/data/vocabularies/date_types.yaml#L21